lib/scoped_search/query_builder.rb

```diff
@@ -485,6 +489,16 @@ def to_sql(builder, definition, &block)
             raise ScopedSearch::QueryNotSupported, "Don't know how to handle this operator node: #{operator.inspect} with #{children.inspect}!"
           end
         end
+
+        private
+
+        def validate_value(field, value)
+          validator = field.validator
+          if field.validator && field.only_explicit
```

Comment:
> We can throw an exception only in case the field is referred explicitly. The method will be called from default fields query too, and there we don't need to validate (and we can't throw an exception).

Are you sure? `validate_value` is only called from `to_single_field_sql` so I think it's only run when a user explicitly searches for the field (which can happen even if `only_explicit` is false).

It isn't called from `to_default_fields_sql` so there's no validation of the value if you search for text without a `field = ` search.